internal/tofu/tofu.go

```diff
@@ -125,6 +125,10 @@ func (t *Tofu) Apply(ctx context.Context) error {
 func (t *Tofu) Destroy(ctx context.Context) error {
 	args := []string{"destroy", "-parallelism", strconv.Itoa(t.threads), "-auto-approve"}
 
+	for _, variable := range t.variables {
```

Comment:
Can we move this "variable parsing" code to a private helper function and just call it in Apply() and Destroy()?